lib/options/option_validator.rb

```diff
@@ -32,25 +33,34 @@ def validate(options)
 
     # Validates BrowserStack options
     def validate_bs(options, errors)
-      # App
-      app = options[Maze::Option::APP]
-      errors << "--#{Maze::Option::APP} must be specified" if app.nil?
-      unless app.start_with? 'bs://'
-        errors << "app file '#{app}' not found" unless File.exist? app
-      end
-
       # BS local binary
       bs_local = options[Maze::Option::BS_LOCAL]
       errors << "BrowserStack local binary '#{bs_local}' not found" unless File.exist? bs_local
 
       # Device
+      bs_browser = options[Maze::Option::BS_BROWSER]
       bs_device = options[Maze::Option::BS_DEVICE]
-      if bs_device.nil?
-        errors << "--#{Maze::Option::BS_DEVICE} must be specified"
-      else
+      if bs_browser.nil? && bs_device.nil?
+        errors << "Either --#{Maze::Option::BS_BROWSER} or --#{Maze::Option::BS_DEVICE} must be specified"
+      elsif !bs_browser.nil?
```

Comment:
It might be a bit clearer to test the values themselves; only `nil` & `false` are falsy

```suggestion
      elsif bs_browser
```

Reply:
Changed as suggested.